ui/mirage/scenarios/default.js

```diff
@@ -1215,3 +1217,117 @@ function getScenarioQueryParameter() {
   return mirageScenario;
 }
 /* eslint-enable */
+
+export function createRestartableJobs(server) {
```

Comment:
Sets jobs/versions up both in our mocked mirage smallCluster (where we can preview this), but also in a way that our tests can leverage it easily via import.